src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -56,6 +56,28 @@ public async Task BulkDocs()
                 }
                 else
                     ThrowNotSupportedType(contentType);
+                if (TrafficWatchManager.HasRegisteredClients)
+                {
+                    var sb = new StringBuilder();
+                    for (var i = 0; i < command.ParsedCommands.Count; i++)
+                    {
+                        // watching only patch commands
+                        if (command.ParsedCommands.Array[i].Patch != null)
+                        {
+                            // reconstruct in json format
+                            sb.Append("{\"").Append(command.ParsedCommands.Array[i].Type).Append("\":{");
+                            sb.Append("\"Script\":\"").Append(command.ParsedCommands.Array[i].Patch.Script).Append("\"");
```

Comment:
Let's NOT use json as the format here. You are not escaping the output.
Just write this in human readable form.

Reply:
done